core/src/main/scala/kafka/server/BrokerToControllerChannelManager.scala

```diff
@@ -211,7 +211,7 @@ class BrokerToControllerChannelManagerImpl(
         50,
         Selectable.USE_DEFAULT_BUFFER_SIZE,
         Selectable.USE_DEFAULT_BUFFER_SIZE,
-        config.requestTimeoutMs,
+        config.controllerSocketTimeoutMs,
```

Comment:
This change is cosmetic right now.  `BrokerToControllerChannelManager` does not leverage this default value.